pkgs/development/python-modules/mpv-jsonipc/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, fetchFromGitHub, requests
+, tqdm, websocket_client, pytest, pillow, mock, isPy27 }:
+
+buildPythonPackage rec {
+  pname = "mpv-jsonipc";
+  version = "unstable-2020-02-01";
+
+  src = fetchFromGitHub {
+    owner = "iwalton3";
+    repo = "python-mpv-jsonipc";
+    rev = "705b977ce4154ef018a43340feaa59fd1b0823f1";
+    sha256 = "1rxp1z4j2agn13n1by32d9v29abmba1zhfj11yywavsjnj64c1ps";
+  };
+
+  # doesn't include tests
+  doCheck = false;
+
+  propagatedBuildInputs = [ requests tqdm websocket_client ];
+
+  checkInputs = [ pytest pillow ]
+    ++ lib.optionals isPy27 [ mock ];
```

Comment:
I think you can inspect the version of `python` via some attribute of it. So please try and use that and don't import `isPy27`.

Reply:
this is fine, isPy27 comes from the python package set, and will be set accordingly.